plugins/modules/redfish_command.py

```diff
@@ -281,6 +281,12 @@
       - BIOS attributes that needs to be verified in the given server.
     type: dict
     version_added: 6.4.0
+  factory_reset_mode:
```

Comment:
I'd rename this since there are end user use cases to reset their BMC to default settings.

Reply:
done